### PR TITLE
Changes to be able to build as Java 11 project

### PR DESCRIPTION
**Pull request by obfischer**
Changes to the project to be able to run it as Java 11 application.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -61,10 +62,24 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-javadoc-plugin</artifactId>
+                <configuration>
+                    <sourcepath>${project.build.directory}/generated-sources/delombok;${project.build.directory}/generated-sources/antlr4;${project.build.directory}/generated-sources/xjc</sourcepath>
```
Comment (StephanPirnbaum) [inline pom.xml:69]: Additional config should not be provided by common lugin, antlr is very special and not used by many plugins

File: pom.xml [hunk 2]
```diff
@@ -61,10 +62,24 @@
...
+                </configuration>
+            </plugin>
+
         </plugins>
     </build>
 
     <dependencies>
+        <dependency>
```
Comment (StephanPirnbaum) [inline pom.xml:77]: Why was this added?

File: pom.xml (hunk 1 above)
Comment (obfischer) [reply pom.xml:68]: You are right. I will remove it. C&P mistake.

File: pom.xml [hunk 3]
```diff
@@ -61,6 +61,14 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```
Comment (DirkMahler) [inline pom.xml:64]: Isn't there already the same config in the uber parent?

Comment (obfischer) [reply pom.xml:64]: Könntet ihr bitte unter der entsprechenden Stelle kommentieren. Ich sehe die Anmerkung am Tag `<plugin>` und nicht das, worauf sich die Anmerkung bezieht.

Comment (obfischer) [reply pom.xml:64]: No, in Uber Parent, the setting is the following `<sourcepath>${project.build.directory}/generated-sources/delombok</sourcepath>`.

Comment (DirkMahler) [reply pom.xml:64]: The idea is to have this for both delombok and xjc specified once in the uber parent and nowhere else. That pom defines both JAXB code generation and Lombok as allowed frameworks for jQA and thus it makes sense to have the appropriate Javadoc config there as well.

Comment (obfischer) [reply pom.xml:64]: Ok, now I got it. I will implement it, as you suggested.

Comment (obfischer) [reply pom.xml:64]: Removed, commited and pushed.

Comment (StephanPirnbaum) [reply pom.xml:64]: It’s not removed here (at least as GitHub shows it to me)

**Review by StephanPirnbaum — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 3 above)
Comment (obfischer) [reply pom.xml]: Updated and removed.

**Review by StephanPirnbaum — APPROVED** [review]
